docs/getting-started/index.md

```diff
@@ -0,0 +1,180 @@
+A Tentacle Agent isn't required for deploying to Azure Web Apps, Kubernetes clusters, Azure Service Fabric, or Azure Cloud Service targets.
+
+**[Lifecycles](/docs/deployment-process/lifecycles/index.md)**
+
+Lifecycles give you control over the way releases are promoted between environments.
+:::
+
+## 3. Package and Upload Your Software
+
+Before you can deploy software with Octopus Deploy, you need to bundle all the files required for the software to run into a supported package. The package must be versioned and stored in a repository. Octopus Deploy includes a built-in repository and supports external package feeds.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ujMBpvQUOrg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### **Packaging Software** (0:56)
+
+1. Give your package a [package ID](/docs/packaging-applications/index.md#package-id).
+1. Choose and apply a [versioning scheme](/docs/packaging-applications/index.md#version-numbers).
+1. Create the package in a [supported format](/docs/packaging-applications/index.md#supported-formats).
+
+### **Uploading a Package** (1:28)
+
+1. Navigate to **{{Library,Packages}}** and click **UPLOAD PACKAGE**.
+1. Select your package and click **SAVE**.
+
+:::hint
+**New to Octopus and don’t have a package?**
+
+Use our example package to quickly create your first deployment. [hello-world.1.0.0.zip](https://octopus.com/images/docs/hello-world.1.0.0.zip)
+:::
+
+:::secondary
+**Learn More**
+
+**[Packaging Your Applications](/docs/packaging-applications/index.md)**
+
+A package is an archive (zip, tar, NuGet) that contains all the files needed to run your software. You can host packages in external repositories or the built-in Octopus repository.
+
+There are many more tools you might choose to use to create your package, but as long as you can create one of our [supported packages](/docs/packaging-applications/index.md#supported-formats) you can deploy your applications with Octopus Deploy.
+
+**[Build Server Integration](/docs/packaging-applications/build-servers/index.md)**
+
+Most Octopus users automate their existing tool chain to push packages to their Octopus Deploy server with our [API and Integrations](/docs/octopus-rest-api/index.md). But you can manually upload the package or host it in an external repository.
+:::
+
+## 4. Define Your Deployment Process
+
+Octopus Deploy is designed to work with teams following agile software development methodologies, that is, continuously deploying software, iterating, making changes, and redeploying. Before you can deploy, you need to create a project with your deployment process. The deployment process contains all the information needed to have your teams successfully redeploy every time.
```

Comment:
"successfully redeploy every time" should be "successfully deploy every time". It's good to mention it in the first sentence so they know they can re-deploy if they need, but we shouldn't be promoting re-deploying in a getting started guide.